kernel/kselftest.py

```diff
@@ -32,6 +32,8 @@ class kselftest(Test):
 
     :see: https://www.kernel.org/doc/Documentation/kselftest.txt
     :source: https://github.com/torvalds/linux/archive/master.zip
+
+    :avocado: tags=kernel,power,privileged
```

Comment:
all the test inside tools/testing/selftest/powerpc  are specific to power like perf, pmu, ebb, radix mmu,etc